dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.12.5-alpine
+
+RUN apk --update add openjdk8-jre-base git
+
+WORKDIR /usr/scr/app
+
+COPY requirements.txt ./
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+EXPOSE 8000
+
+VOLUME ["/tmp_home", "/persistent_home"]
+
+ENV DJANGO_TMP_HOME="/tmp_home"
+
+ENV DJANGO_PERSISTENT_HOME="/persistent_home"
+
+CMD ["gunicorn", "--bind", "0.0.0.0:8000", "xlsform_prj.wsgi:application"]
```

Comment:
Validate/Javarosa historically has a memory leak so I use jitter to resolve that in production. See https://github.com/getodk/xlsform-server/blob/master/provisioning/roles/xlsform/templates/gunicorn.service.j2#L10